moped-editor/src/components/DataGridPro/DataGridActions.js

```diff
@@ -45,8 +45,12 @@ const DataGridActions = ({
    */
   const hasRequiredFields = useGridSelector(apiRef, () => {
     const editState = apiRef.current.state.editRows;
+
     for (const field of requiredFields) {
-      if (!editState[id]?.[field]?.value) {
+      const hasError = editState[id]?.[field]?.error;
```

Comment:
this `error` state comes from its definition in [preProcessEditCellProps](https://mui.com/x/react-data-grid/editing/?srsltid=AfmBOorAjy4Egs9sAYiz-XwgY_B6xVw3agN4JORVCgeFvxCLA5VKK3q5#validation) in the column config coming out of the `useColumns` hook.

Further down in the diff, you can see the check for the team role value having an array length greater than 0.